Add reference to scrape interval in the K8s helm in stallation doc (#11695)
newrelic/docs-website: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

  # Other options from https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure-v3
```

> The full list of flags (e.g., [scrape-interval](https://docs.newrelic.com/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-optimize-ingest-guide/#k8s-integration)) that can be tweaked can be found in our chart's repository:

Reviewer: ```suggestion
The full list of flags (such as [`scrape-interval`](/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-optimize-ingest-guide#k8s-integration)) that can be tweaked can be found in our chart's repository:
```
